packages/react-search-ui/src/WithSearchRenderProps.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import withSearch from "./withSearch";
+import PropTypes from "prop-types";
+
+function WithSearch({ mapContextToProps: mapContextToProps, children }) {
```

Comment:
What does this part mean: "mapContextToProps: mapContextToProps"?

Reply:
@yakhinvadim Another find / replace typo, something my editor did, will clean it up and double check for any more errors. TY.